sopel/plugin.py

```diff
@@ -913,11 +914,14 @@ def url(*url_rules):
         def handle_example_bugs(bot, trigger, match):
             bot.reply('Found bug ID #%s' % match.group(1))
 
-    This should be used rather than the matching in ``trigger``, in order to
-    support e.g. the ``.title`` command.
+    Both ``trigger`` and ``match`` represent the same match on the URL. The
+    ``match`` object is a Python built-in :func:`match object <re.match>`, while
```

Comment:
See how it's intersphinx'd in `Trigger`: https://github.com/sopel-irc/sopel/blob/b022f461e2a8f321c53fe508a78facf8846c3e0d/sopel/trigger.py#L187

and how it appears in [the current live docs](https://sopel.chat/docs/trigger.html#sopel.trigger.Trigger) for `Trigger`'s constructor.

```suggestion
    ``match`` object is a Python built-in :ref:`match object <match-objects>`, while
```